<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>merge_conflict</class>
 <widget class="QDialog" name="merge_conflict">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>371</width>
     <height>391</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>369</width>
      <height>389</height>
     </rect>
    </property>
    <widget class="QPushButton" name="filter_employees_bt">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>30</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Filter</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="add_employee_bt">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>160</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Employee</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_employee_bt">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>420</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete Employee</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Employee List</string>
   </property>
  </widget>
  <widget class="QPushButton" name="filter_employees_bt">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>140</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Filter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="archive_employee_bt">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>210</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Archive Employee</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Status</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>40</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Position</string>
   </property>
  </widget>
  <widget class="QPushButton" name="temp_employee_bt">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>290</y>
     <width>160</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Temp Employee</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
